### ISSUE TITLE
Does this work with code mirror editor?

### ISSUE THREAD

**Issue by linonetwo** [snowgoon88/TW5-extendedit#15]
It seems not working on my side.

I'm in 5.1.22

**Comment by linonetwo** [comment]
I see...

> in $:/ControlPanel -> Advanced -> Editor Type -> text/vnd.tiddlywiki you must click it, and change the content of poped tiddler from `codemirror` to `comptext`.

**Comment by snowgoon88** [comment]
Yes, sorry, but in fact this completion works only with basic/simple editor. In fact, I recreate the editor and add the completion/popup addon to it. I do not know codemirror to see if a version with completion could be possible.
I reopen to remind me to have look. One day...

**Comment by atlas-comstock** [comment]
@snowgoon88 Hi, can you check this again? I really want your plugin works with CodeMirror, thanks.
Your plugin is sooo good, thanks for your work.

**Comment by snowgoon88** [comment]
Hello,
There are 2 BIG difficulties I see

1) how to integrate the completion capability to the code mirror editor of a Tiddler. It might be that the work of saqimtiaz could help, but I would have to explore and check. A bit time consuming.

2) more importantly. What kind of completion would you need. If it not linked to the title of another Tiddler, my module will not be easy to adapt. DO NOT expect me to make a syntaxic or lexical analysis of the javascript inside tiddler to offer completion of variables or function or...

So, before i invest some time in this, I would like you to precise what you expect of the plugin working "in code mirror".

Alain

**Comment by saqimtiaz** [comment]
Note that there are working solutions for title autocompletion in code mirror.

I posted a quick proof of concept last year. There is a code mirror plus add-on that does something similar: https://groups.google.com/g/tiddlywiki/c/_61Wvbkph9w/m/jVvOAUFvBwAJ

The core editor has a button for inserting links to tiddlers, which as of 5.1.23 has complete keyboard support and is triggered by ctrl+L

I don't think that extending this plugin to work with CodeMirror makes sense or is needed.

**Comment by atlas-comstock** [comment]
@snowgoon88 @saqimtiaz 

Thanks for the quick reply.

I like using TW5-extendedit to insert internal links ([[some topic]]), however, the complex type file editor is so simple/plain.
So, I want to use CodeMirror with the function that TW5-extendedit  can provide.

@saqimtiaz  I had tried the auto complete of TW5-CodeMirror-Plus, it is so simple, does not support other TW title complement.

**Comment by saqimtiaz** [comment]
It is a significantly less work and a more logical approach to ask to extend the existing CodeMirror autocomplete.

**Comment by atlas-comstock** [comment]
@saqimtiaz Indeed. Then let me close this issue, thank you guys.
